inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -372,6 +372,9 @@ def execute_from_manifest(self,
                 self._dispatcher_broker.send_result(SOTA_FAILURE)
                 raise SotaError(SOTA_FAILURE)
```

Comment:
**Suggestion:** The execute_from_manifest method raises a SotaError with a generic failure message. It would be beneficial to include more context in the error message to aid in debugging. Consider appending additional information such as the state or command that failed. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                error_message = f"{SOTA_FAILURE} during state: {self.sota_state} with command: {self.sota_cmd}"
                self._dispatcher_broker.send_result(error_message)
                raise SotaError(error_message)
```

Reply:
Changing our error messages from what is expected causes issues for validation.